Src/PerformanceCollector/Perf.NetFull.Tests/Filtering/CollectionConfigurationTests.cs

```diff
@@ -1,4 +1,4 @@
-﻿namespace Unit.Tests
+﻿namespace Microsoft.ApplicationInsights
```

Comment:
Please do not pollute root namespace and make it somehow matching the project name